app/views/providers/submitted_applications/show.html.erb

```diff
@@ -11,7 +11,7 @@
     </dd>
   </dl>
 
-  <button class="govuk-button">
+  <button class="govuk-button no-print" onclick="window.print()">
```

Comment:
I think this is crying out to be put into a helper method:
```ruby
def print_button
  content_tag :button, t('generic.print_button'), class: 'govuk-button no-print', onclick: 'window.print()'
end
```
That way the print functionality is separated from the markup.

Then here:
```
<%= print_button %>
```
Note the movement of the label translation to generic. Alternatively the label could be passed in:
```
<%= print_button t('.print_button') %>
```